src/style.css

```
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+
+}
+
+.search {
+  margin-top: 20px;
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.icon__small {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.input::placeholder {
+  font-weight: 300px;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.small {
```

or something that indicates that `small` is related to input
```suggestion
.input-small {
```